OrbitProducerSideChannel/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+project(OrbitProducerSideChannel)
+
+add_library(OrbitProducerSideChannel INTERFACE)
+target_compile_options(OrbitProducerSideChannel INTERFACE ${STRICT_COMPILE_FLAGS})
+
+target_sources(OrbitProducerSideChannel INTERFACE
+        include/OrbitProducerSideChannel/ProducerSideChannel.h)
+
+target_include_directories(OrbitProducerSideChannel INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
@beckerhe, should I add 

```
target_link_libraries(OrbitProducerSideChannel INTERFACE
        CONAN_PKG::abseil)
```

here? And if yes, what about gRPC for `#include <grpcpp/grpcpp.h>`? What do I need to link for that?

Reply:
You should also add `CONAN_PKG::grpc`.

Reply:
Ah well, `grpc` is obviously what I need here, not `protobuf`. Whoops. Done.